Frontend/Gmfctn/src/app/modules/+auth/components/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, EventEmitter, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { finalize, take } from 'rxjs/operators';
+import { AuthenticateService } from '../../services/authenticate.service';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit, OnDestroy {
+  @Output() isSigned = new EventEmitter<boolean>();
+  state = '';
+  loading = false;
+  subscribtion$: Subscription = new Subscription;
+  subscribtionForCorrectness$: Subscription = new Subscription;
```

Comment:
This usage of subscriptions is incorrect. 
1. you don't need two subscriptions
2. you should instantiate is like `subscriptions = new Subscription(); `
3. in the component it's used as `this.subscriptions.add(observable$)`
4. To unsubscribe in ngOnDestroy you should call `this.subscriptions.unsubscribe();` once, no matter how much subscriptions you have added there

Also you have a typo: subscriPtion not subscriBtion)

Reply:
Fixed